src/client/components/Home.jsx

```diff
@@ -22,7 +22,17 @@ export default class Home extends React.Component {
   }
 
   async fetchRecipes () {
-    const recipes = await recipeFetcher.getRecipes(this.recipeQuery.value, 0, 10);
+    let recipes = await recipeFetcher.getRecipes(this.recipeQuery.value, 0, 10);
+    let recipesLen = recipes.length;
```

Comment:
You could just use the [Array.sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) method like this:
```
recipes = recipes.sort((recipeA, recipeB) =>{
    return recipeA.recipe.calories - recipeB.recipe.calories;
});
```
but besides that, looks good.